src/Commands/CreateConfigCommand.php

```diff
@@ -168,7 +160,7 @@ public function createConfig($file, array $options = ['create-content' => FALSE,
       $this->io()->comment($this->contentCreator->getOutputDirectory());
```

Comment:
What about these comments here?

Reply:
Good point. Tidied them up and some other things.